data_management/opensearch_indexer/opensearch_indexer/text_extraction.py

```diff
@@ -8,6 +10,8 @@
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
+CONVERT_MAP = {"xls": "xlsx", "doc": "docx"}
```

Comment:
maybe worth making this constant name a little more descriptive
```suggestion
TEXTRACT_FILE_FORMAT_FALLBACK_CONVERSION_MAP = {"xls": "xlsx", "doc": "docx"}
```

word